data/darktableconfig.xml.in

```diff
@@ -557,7 +557,7 @@
     <type>bool</type>
     <default>true</default>
     <shortdescription>sort collection recent to older</shortdescription>
-    <longdescription>changes the default collections-list order for folders, times and dates to run from recent to older</longdescription>
+    <longdescription>changes the default collections-list order for film rolls, folders, times and dates to run from recent to older</longdescription>
```

Comment:
we should change the name and description of this, possibly to just something like "sort descending" or "sort ascending" depending on which way we want the logic to work. "Recent to older" doesn't really make sense when sorting on text fields.